Makefile

```diff
@@ -136,7 +136,8 @@ centos8-nvidia: ## Build Centos 8 image
 .PHONY: flatcar
 flatcar: build
 flatcar: ## Build flatcar image
-	./bin/konvoy-image build images/ami/flatcar.yaml
+	AWS_DEFAULT_REGION=$(AWS_REGION) ./hack/fetch-flatcar-ami.sh
```

Comment:
Should we make this a separate make target that creates `flatcar-version.yaml`.

Reply:
On one hand, that saves fetching the data repeatedly. On the other hand, it may mean we attempt to build an older version when a new version becomes available. But it would make the e2e test cleaner, since I just added the same command there (although the e2e tests seems to pass without it).

Reply:
Changed to use a separate targer